test/spec/controllers/exportimport.js

```diff
@@ -210,7 +203,7 @@ describe('Controller: ExportImportCtrl', function () {
 
     scope.createExportFile( scope.selectedExports );
 
-    expect(scope.downloadLink.indexOf('blob:http://localhost:8080/')).to.be.above(-1);
+    expect(scope.downloadLink.indexOf('')).to.be.above(-1);
```

Comment:
This is something that came up when we upgraded the packages last week. Older versions of phantom didn't support blobs, the newer version does. Please try deleting your node_modules folder, npm install and then running the tests again?